### PR TITLE
Add Plausible Analytics docs to showcase page

### PR DESCRIPTION
**Pull request by metmarkosaric**
## Motivation

One of the Docusaurus maintainers mentioned the showcase on Twitter so wanted to add Plausible to the list as we're using Docusaurus v2 for our docs

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Run yarn run start:v2 and check that the page http://localhost:3000/showcase is rendered properly.

## Related PRs

None

### CODE REVIEW

File: website/src/data/users.js [hunk 1]
```diff
@@ -707,6 +707,14 @@ const Users = [
     source: null,
     tags: ["product"],
   },
+  {
+    title: 'Plausible Analytics Docs',
+    description: 'Plausible Analytics is a simple, open source, lightweight (< 1 KB) and privacy-friendly alternative to Google Analytics. ',
+    preview: require('./showcase/plausible.png'),
```
Comment (lex111) [inline website/src/data/users.js:713]: @metmarkosaric it looks like you forgot to commit screenshot of the site. Can you fix that?

**Comment by slorber** [comment]
thanks 👍

**Comment by metmarkosaric** [comment]
thank you @slorber!

**Comment by svrooij** [comment]
@metmarkosaric Such a shame that you seem to use docusaurus for the Plausible documentation, but there is no official support for plausible in docusaurus.

I guess you must be using it already, did you create a custom docusaurus plugin?

**Comment by metmarkosaric** [comment]
@svrooij! Hey! We have instructions here https://plausible.io/docs/docusaurus-integration

**Comment by svrooij** [comment]
@metmarkosaric I could not find it in the documentation, so I opened the source of your own docs and found [this code](outbound-links), which helped 🎉

**Pull request merged**